java/ql/src/experimental/Security/CWE/CWE-470/LoadClassNoSignatureCheck.ql

```diff
@@ -20,7 +20,7 @@ import semmle.code.java.dataflow.TaintTracking
 
 MethodAccess getClassLoaderReachableMethodAccess(DataFlow::Node node)
 {
-    exists(MethodCall maGetClassLoader |
+    exists(MethodAccess maGetClassLoader |
```

Comment:
I need to change MethodCall back to MethodAccess. Otherwise, codeql complains:
`Error: Failed to run query: ERROR: Could not resolve type MethodCall`